apps/condo/domains/ticket/access/TicketContact.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Generated by `createschema ticket.TicketContact 'property:Relationship:Property:PROTECT; unitName:Text; email:Text; phone:Text; name:Text;' --force`
+ */
+const { checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
+const get = require('lodash/get')
+const { getById } = require('@core/keystone/schema')
+
+async function canReadTicketContacts ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) return {}
+    return {
+        property: { organization: { employees_some: { user: { id: user.id }, isBlocked: false } } },
```

Comment:
it's better to connect directly to an organization in such cases to reduce the query complexity

Reply:
I will reduce query complexity, but will increase code complexity, because we will need to add validation for connected `Property` and `Organization` to be associated with each other, to prevent cases, where connected `Property` is belonging to another connected `Organization`.
Is this critical currently?